packages/llvm-mingw-w64/build.sh

```diff
@@ -28,7 +28,6 @@ termux_step_make_install() {
 
 	# On Termux, use the wrapper script rather than the wrapper binary
 	rm *wrapper
-	rm *wrapper.sh.orig
```

Comment:
Sometimes `patch -p1` will generate this file. It should be `rm *wrapper.sh.orig || true`